File: RAT/Hades v1.6/Source/xRAT.vb/xRAT/Stealer.vb
Imports Microsoft.VisualBasic.CompilerServices
Imports System
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Drawing
Imports System.Runtime.CompilerServices
Imports System.Windows.Forms

Namespace xRAT
    <DesignerGenerated> _
    Public Class Stealer
        Inherits Form
        ' Methods
        Public Sub New()
            Class1.QaIGh5M7cuigS
            AddHandler MyBase.Load, New EventHandler(AddressOf Me.Stealer_Load)
            Me.InitializeComponent
        End Sub

        <DebuggerNonUserCode> _
        Protected Overrides Sub Dispose(ByVal disposing As Boolean)
            Try 
                If (disposing AndAlso (Not Me.icontainer_0 Is Nothing)) Then
                    Me.icontainer_0.Dispose
                End If
            Finally
                MyBase.Dispose(disposing)
            End Try
        End Sub

        Private Sub exitbtn_Click(ByVal sender As Object, ByVal e As EventArgs)
            Me.Close
        End Sub

        <DebuggerStepThrough> _
        Private Sub InitializeComponent()
            Me.icontainer_0 = New Container
            Dim manager As New ComponentResourceManager(GetType(Stealer))
            Me.PictureBox1 = New PictureBox
            Me.Label7 = New Label
            Me.exitbtn = New Button
            Me.Timer1 = New Timer(Me.icontainer_0)
            Me.RichTextBox1 = New RichTextBox
            DirectCast(Me.PictureBox1, ISupportInitialize).BeginInit
            Me.SuspendLayout
            Me.PictureBox1.Image = Class5.smethod_49
            Dim point As New Point(11, &HF1)
            Me.PictureBox1.Location = point
            Me.PictureBox1.Name = "PictureBox1"
            Dim size As New Size(&H10, &H10)
            Me.PictureBox1.Size = size
            Me.PictureBox1.SizeMode = PictureBoxSizeMode.AutoSize
            Me.PictureBox1.TabIndex = &H16
            Me.PictureBox1.TabStop = False
            Me.Label7.AutoSize = True
            point = New Point(&H21, &HF2)
            Me.Label7.Location = point
            Me.Label7.Name = "Label7"
            size = New Size(&H49, 13)
            Me.Label7.Size = size
            Me.Label7.TabIndex = &H15
            Me.Label7.Text = "Please wait..."
            point = New Point(&HFD, &HEC)
            Me.exitbtn.Location = point
            Me.exitbtn.Name = "exitbtn"
            size = New Size(&H4E, &H18)
            Me.exitbtn.Size = size
            Me.exitbtn.TabIndex = 20
            Me.exitbtn.Text = "&Close"
            Me.exitbtn.UseVisualStyleBackColor = True
            point = New Point(12, 12)
            Me.RichTextBox1.Location = point
            Me.RichTextBox1.Name = "RichTextBox1"
            size = New Size(&H13F, &HDA)
            Me.RichTextBox1.Size = size
            Me.RichTextBox1.TabIndex = &H18
            Me.RichTextBox1.Text = ""
            Dim ef As New SizeF(6!, 13!)
            Me.AutoScaleDimensions = ef
            Me.AutoScaleMode = AutoScaleMode.Font
            Me.BackColor = Color.White
            size = New Size(&H157, &H110)
            Me.ClientSize = size
            Me.Controls.Add(Me.RichTextBox1)
            Me.Controls.Add(Me.PictureBox1)
            Me.Controls.Add(Me.Label7)
            Me.Controls.Add(Me.exitbtn)
            Me.FormBorderStyle = FormBorderStyle.Fixed3D
            Me.Icon = DirectCast(manager.GetObject("$this.Icon"), Icon)
            Me.Name = "Stealer"
            Me.StartPosition = FormStartPosition.CenterScreen
            Me.Text = "Stealer"
            DirectCast(Me.PictureBox1, ISupportInitialize).EndInit
            Me.ResumeLayout(False)
            Me.PerformLayout
        End Sub

        Private Sub Stealer_Load(ByVal sender As Object, ByVal e As EventArgs)
            Me.Timer1.Start
            Me.PictureBox1.Visible = True
            Me.Label7.Visible = True
        End Sub

        Private Sub Timer1_Tick(ByVal sender As Object, ByVal e As EventArgs)
            If (Me.RichTextBox1.Text = "") Then
                Me.RichTextBox1.Text = Class2.Class4_0.method_6.RichTextBox2.Text
            Else
                Me.Label7.Visible = False
                Me.PictureBox1.Visible = False
                Me.Timer1.Stop
            End If
        End Sub


        ' Properties
        Friend Overridable Property exitbtn As Button
            Get
                Return Me._exitbtn
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As Button)
                Dim handler As EventHandler = New EventHandler(AddressOf Me.exitbtn_Click)
                If (Not Me._exitbtn Is Nothing) Then
                    RemoveHandler Me._exitbtn.Click, handler
                End If
                Me._exitbtn = value
                If (Not Me._exitbtn Is Nothing) Then
                    AddHandler Me._exitbtn.Click, handler
                End If
            End Set
        End Property

        Friend Overridable Property Label7 As Label
            Get
                Return Me._Label7
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As Label)
                Me._Label7 = value
            End Set
        End Property

        Friend Overridable Property PictureBox1 As PictureBox
            Get
                Return Me._PictureBox1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As PictureBox)
                Me._PictureBox1 = value
            End Set
        End Property

        Friend Overridable Property RichTextBox1 As RichTextBox
            Get
                Return Me._RichTextBox1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As RichTextBox)
                Me._RichTextBox1 = value
            End Set
        End Property

        Friend Overridable Property Timer1 As Timer
            Get
                Return Me.timer_0
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As Timer)
                Dim handler As EventHandler = New EventHandler(AddressOf Me.Timer1_Tick)
                If (Not Me.timer_0 Is Nothing) Then
                    RemoveHandler Me.timer_0.Tick, handler
                End If
                Me.timer_0 = value
                If (Not Me.timer_0 Is Nothing) Then
                    AddHandler Me.timer_0.Tick, handler
                End If
            End Set
        End Property


        ' Fields
        <AccessedThroughProperty("exitbtn")> _
        Private _exitbtn As Button
        <AccessedThroughProperty("Label7")> _
        Private _Label7 As Label
        <AccessedThroughProperty("PictureBox1")> _
        Private _PictureBox1 As PictureBox
        <AccessedThroughProperty("RichTextBox1")> _
        Private _RichTextBox1 As RichTextBox
        Private icontainer_0 As IContainer
        <AccessedThroughProperty("Timer1")> _
        Private timer_0 As Timer
    End Class
End Namespace

File: RAT/Hades v1.6/Source/xRAT.cs/xRAT/Stealer.cs
namespace xRAT
{
    using Microsoft.VisualBasic.CompilerServices;
    using System;
    using System.ComponentModel;
    using System.Diagnostics;
    using System.Drawing;
    using System.Runtime.CompilerServices;
    using System.Windows.Forms;

    [DesignerGenerated]
    public class Stealer : Form
    {
        [AccessedThroughProperty("exitbtn")]
        private Button _exitbtn;
        [AccessedThroughProperty("Label7")]
        private Label _Label7;
        [AccessedThroughProperty("PictureBox1")]
        private PictureBox _PictureBox1;
        [AccessedThroughProperty("RichTextBox1")]
        private RichTextBox _RichTextBox1;
        private IContainer icontainer_0;
        [AccessedThroughProperty("Timer1")]
        private Timer timer_0;

        public Stealer()
        {
            Class1.QaIGh5M7cuigS();
            base.Load += new EventHandler(this.Stealer_Load);
            this.InitializeComponent();
        }

        [DebuggerNonUserCode]
        protected override void Dispose(bool disposing)
        {
            try
            {
                if (disposing && (this.icontainer_0 != null))
                {
                    this.icontainer_0.Dispose();
                }
            }
            finally
            {
                base.Dispose(disposing);
            }
        }

        private void exitbtn_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        [DebuggerStepThrough]
        private void InitializeComponent()
        {
            this.icontainer_0 = new System.ComponentModel.Container();
            ComponentResourceManager manager = new ComponentResourceManager(typeof(Stealer));
            this.PictureBox1 = new PictureBox();
            this.Label7 = new Label();
            this.exitbtn = new Button();
            this.Timer1 = new Timer(this.icontainer_0);
            this.RichTextBox1 = new RichTextBox();
            ((ISupportInitialize) this.PictureBox1).BeginInit();
            this.SuspendLayout();
            this.PictureBox1.Image = Class5.smethod_49();
            Point point = new Point(11, 0xf1);
            this.PictureBox1.Location = point;
            this.PictureBox1.Name = "PictureBox1";
            Size size = new Size(0x10, 0x10);
            this.PictureBox1.Size = size;
            this.PictureBox1.SizeMode = PictureBoxSizeMode.AutoSize;
            this.PictureBox1.TabIndex = 0x16;
            this.PictureBox1.TabStop = false;
            this.Label7.AutoSize = true;
            point = new Point(0x21, 0xf2);
            this.Label7.Location = point;
            this.Label7.Name = "Label7";
            size = new Size(0x49, 13);
            this.Label7.Size = size;
            this.Label7.TabIndex = 0x15;
            this.Label7.Text = "Please wait...";
            point = new Point(0xfd, 0xec);
            this.exitbtn.Location = point;
            this.exitbtn.Name = "exitbtn";
            size = new Size(0x4e, 0x18);
            this.exitbtn.Size = size;
            this.exitbtn.TabIndex = 20;
            this.exitbtn.Text = "&Close";
            this.exitbtn.UseVisualStyleBackColor = true;
            point = new Point(12, 12);
            this.RichTextBox1.Location = point;
            this.RichTextBox1.Name = "RichTextBox1";
            size = new Size(0x13f, 0xda);
            this.RichTextBox1.Size = size;
            this.RichTextBox1.TabIndex = 0x18;
            this.RichTextBox1.Text = "";
            SizeF ef = new SizeF(6f, 13f);
            this.AutoScaleDimensions = ef;
            this.AutoScaleMode = AutoScaleMode.Font;
            this.BackColor = Color.White;
            size = new Size(0x157, 0x110);
            this.ClientSize = size;
            this.Controls.Add(this.RichTextBox1);
            this.Controls.Add(this.PictureBox1);
            this.Controls.Add(this.Label7);
            this.Controls.Add(this.exitbtn);
            this.FormBorderStyle = FormBorderStyle.Fixed3D;
            this.Icon = (Icon) manager.GetObject("$this.Icon");
            this.Name = "Stealer";
            this.StartPosition = FormStartPosition.CenterScreen;
            this.Text = "Stealer";
            ((ISupportInitialize) this.PictureBox1).EndInit();
            this.ResumeLayout(false);
            this.PerformLayout();
        }

        private void Stealer_Load(object sender, EventArgs e)
        {
            this.Timer1.Start();
            this.PictureBox1.Visible = true;
            this.Label7.Visible = true;
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            if (this.RichTextBox1.Text == "")
            {
                this.RichTextBox1.Text = Class2.Class4_0.method_6().RichTextBox2.Text;
            }
            else
            {
                this.Label7.Visible = false;
                this.PictureBox1.Visible = false;
                this.Timer1.Stop();
            }
        }

        internal virtual Button exitbtn
        {
            get
            {
                return this._exitbtn;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                EventHandler handler = new EventHandler(this.exitbtn_Click);
                if (this._exitbtn != null)
                {
                    this._exitbtn.Click -= handler;
                }
                this._exitbtn = value;
                if (this._exitbtn != null)
                {
                    this._exitbtn.Click += handler;
                }
            }
        }

        internal virtual Label Label7
        {
            get
            {
                return this._Label7;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._Label7 = value;
            }
        }

        internal virtual PictureBox PictureBox1
        {
            get
            {
                return this._PictureBox1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._PictureBox1 = value;
            }
        }

        internal virtual RichTextBox RichTextBox1
        {
            get
            {
                return this._RichTextBox1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._RichTextBox1 = value;
            }
        }

        internal virtual Timer Timer1
        {
            get
            {
                return this.timer_0;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                EventHandler handler = new EventHandler(this.Timer1_Tick);
                if (this.timer_0 != null)
                {
                    this.timer_0.Tick -= handler;
                }
                this.timer_0 = value;
                if (this.timer_0 != null)
                {
                    this.timer_0.Tick += handler;
                }
            }
        }
    }
}

